doc/classes/ProjectSettings.xml

```diff
@@ -1711,6 +1707,18 @@
 		<member name="rendering/reflections/sky_reflections/texture_array_reflections.mobile" type="bool" setter="" getter="" default="false">
 			Lower-end override for [member rendering/reflections/sky_reflections/texture_array_reflections] on mobile devices, due to performance concerns or driver support.
 		</member>
+		<member name="rendering/scaling_3d/fsr_mipmap_bias" type="float" setter="" getter="" default="0.0">
+			Affects the final texture sharpness by reading from a lower or higher mipmap. Negative values make textures sharper, while positive values make textures blurrier. When using FSR, this value is used to adjust the mipmap bias calculated internally which is based on the selected quality. The formula for this is [code]-log2f(1 / quality) + mipmap_bias[/code]
```

Comment:
```suggestion
			Affects the final texture sharpness by reading from a lower or higher mipmap. Negative values make textures sharper, while positive values make textures blurrier. When using FSR, this value is used to adjust the mipmap bias calculated internally which is based on the selected quality. The formula for this is [code]-log2(1.0 / quality) + mipmap_bias[/code]
```
This makes it valid pseudocode for our scripting API.

Reply:
Is this mipmap bias also used when FSR is not enabled? If so, it should most likely be moved out of the `fsr` section.

Also, if it can be changed at run-time, the RenderingServer method for doing so should be mentioned (unless this project setting is read every frame).